src/lib389/lib389/cli_ctl/entryuuid.py

```diff
@@ -0,0 +1,83 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+
+import os
+from lib389.plugins import EntryUUIDPlugin, EntryUUIDSyntaxPlugin
+from lib389.cli_base.dsrc import dsrc_to_ldap, dsrc_arg_concat
+from lib389._constants import DSRC_HOME
+from lib389.cli_base import connect_instance, disconnect_instance
+
+
+def get_connected_inst(inst, log, args):
+    # update the args for connect_instance()
+    args.basedn = None
+    args.binddn = None
+    args.bindpw = None
+    args.starttls = None
+    args.pwdfile = None
+    args.prompt = False
+    new_inst = inst
+    dsrc_inst = dsrc_to_ldap(DSRC_HOME, args.instance, log.getChild('dsrc'))
+    dsrc_inst = dsrc_arg_concat(args, dsrc_inst)
+    try:
+        new_inst = connect_instance(dsrc_inst=dsrc_inst, verbose=args.verbose, args=args)
+    except Exception:
+        # just assume offline
+        pass
+    return new_inst
+
+def install_plugin(inst, log, args):
+    """Add the plugin entries to cn=config"""
+
+    # This can only be done if the plugin package is installed
+    plugin_found = os.system("rpm -qa | grep 389-ds-base-entryuuid")
```

Comment:
This will fail on anything that's not rpm based, so I think we can't do this check in this way.

Reply:
I agree, perhaps we can check for presence of `libentryuuid-plugin.so` and `libentryuuid-syntax-plugin.so` under `get_plugin_dir()`?

Reply:
I'll work on this...